packages/lesmis/src/components/Breadcrumbs.js

```diff
@@ -40,49 +40,36 @@ const Loader = () => (
   </Skeleton>
 );
 
-const getHierarchy = (entities, entityCode, hierarchy = []) => {
-  const entity = entities.find(entity => entity.code === entityCode);
-
-  if (!entity) {
-    return [];
-  }
-
-  const newHierarchy = [entity, ...hierarchy];
-
-  if (entity.type === 'country') {
-    return newHierarchy;
-  }
-  return getHierarchy(entities, entity.parentCode, newHierarchy);
-};
+export const Breadcrumbs = ({ isLoading, breadcrumbs }) => (
+  <StyledBreadcrumbs separator={<NavigateNextIcon />}>
+    <Link to="/">Home</Link>
+    {isLoading ? (
+      <Loader />
+    ) : (
+      breadcrumbs.map(({ name, urlSegment }, index) => {
+        const last = index === breadcrumbs.length - 1;
+        return last ? (
+          <ActiveSegment key={urlSegment}>{name}</ActiveSegment>
+        ) : (
+          <Link to={makeEntityLink(urlSegment)} key={urlSegment}>
```

Comment:
This is the last entity specific part of breadcrumbs - might as well get rid of it now while you're at it?

Reply:
@edmofro - done